packages/jest-config/src/validConfig.js

```diff
@@ -55,9 +55,13 @@ module.exports = ({
   modulePathIgnorePatterns: ['<rootDir>/build/'],
   modulePaths: ['/shared/vendor/modules'],
   name: 'string',
+  noDefaultReporters: false,
```

Comment:
Sounds good. Let's start with Dmitrii's suggestion and only implement "reporters" and then see what that looks like and take it from there :)